### DISCUSSION TITLE
Visor cartográfico - Lluvia de ideas

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#27] [Ideas]
Abro tema para hacer lluvia de ideas en nuestro proto visor (sobre el de IBERPRIX no podemos forzarles a tocar nada)

- Definir niveles de zoom (otros mapant están entre 0 y 9-11)
- Añadir opción de "REPORTAR TILE"
- Exportar PNG / PGW con ajuste a A4 1:5000 / 1:7500 / 1:10000 / 1:15000
- Opción de incluir líneas azules de norte magnético al exportar
- Incluir botón de Leyenda / Simbología Mapant
-

**Comment by manudona92** [comment]
Antes de ayer, Mapant Nueva Zelanda actualizó su visor: http://omap.nz/

- Incluye leyenda y licencia CC BY 4.0 en laparte inferior izquierda

![image](https://user-images.githubusercontent.com/76484741/106201110-42f28980-61b8-11eb-8163-ce00f15eeb74.png)

- Sistema de exportación

**Comment by manudona92** [comment]
Hablando con Mario Vidal, dijimos de alguna forma enlazar desde www.mapant.es a el visor cartográfico de la FEDO. Ellos harían lo mismo, de alguna sección de la página de la FEDO (quizás una propia de Mapant), enlazar con nuestro proto visor. 

La mejor forma que veo de hacer esto es pinchando en el logo de la FEDO (arriba a la derecha en el protovisor), que enlace a: 

https://carto.fedo.org/

**Comment by agustincaballero** [comment]
Dejo aquí algunas sugerencias:

· Poner a la derecha botón de Capas. La capa por defecto podría ser ya la del geotiff.

· Otro botón de menú con: buscador, localización GPS, About, Estadísticas, etc.

· ¿Otro botón Descargar o poner opción Descargar dentro del menú anterior?

· Dejar parte superior para: logotipos, avance del proyecto, y lo que queramos que esté más "a mano"

· En la parte inferior se puede poner lo del copyright, como en NZ

· La capa Geotiff debería ser la capa por defecto; hay que decidir qué mostramos en las zonas donde aún no tengamos nada. ¿Perfil de todo el territorio? ¿Openstreetmap (no sé cómo de factible es esto)?

· Propuesta: procesar las cuadrículas por hoja 50000, cuando estén completas se genera automáticamente (*) el geotiff y ya sería público. 

· La capa Mapant, que permitiría ver las cuadrículas en tiempo real al ser generadas, quedaría sólo para trabajo interno nuestro.

**Comment by manudona92** [comment]
Perfecto. 

- Capa "geotiff" actual por defecto OK. Lo veo óptimo
- Si se utiliza un icono para ABOUT quizá este tipo estaría bien
![image](https://user-images.githubusercontent.com/76484741/110312042-e0857800-8004-11eb-9941-84f6f33738e6.png)
- Si se utiliza icono para localización GPS, quizá utilizaría este, en vez de tu visor. Pero es una sugerencia
![image](https://user-images.githubusercontent.com/76484741/110312132-fdba4680-8004-11eb-9788-36e62f993d26.png)
-Quizá le llamaría "Exportar" en vez de "descarga", así seguimos el mismo nombre que los otros Mapant
-Para las zonas donde no tengamos nada en la capa principal Mapant (geotif), quizá yo pondría con una linea gorda blanca o negra (si el fondo es azul cyan) con el contorno de todo el territorio. Algo que se pueda desactivar y activar (¿?), un layer tipo "Fronteras". Finlandia tiene solo las fronteras continentales, pero no líneas de costa. Pero si nosotros no pusieramos lineas de costa solo tendriamos Pirineos y Portugal y no se puede ver bien en que zona estas. 

Futuros desarrollos (no primordiales en etapa inicial): 
- Muñequito de Street View (arrastrable)
![image](https://user-images.githubusercontent.com/76484741/110292675-df951c00-7fed-11eb-9353-2cc539fd79f3.png)
- Herramienta "Dibujar" (como en tu visor). Permite dibujar solo triángulo, círculo, doble círculo y línea. En magenta con tamaño de símbolo ISOM 2017-2 a 1:10K
- Herramienta "Medir distancia y área". Creo que puede ser muy útil.
![image](https://user-images.githubusercontent.com/76484741/110292750-fc315400-7fed-11eb-864c-2e35ad6d5c44.png)
